components/Column.js

```diff
@@ -1,7 +1,20 @@
 import Row from './Row'
+import { Component } from 'react';
 
-export default ({ children, heading, loading, loadingAmount = 20, className }) => (
-  <div className={`column ${className}`}>
+class Column extends Component {
+  constructor (props) {
```

Comment:
just a heads up, there's no need to put a constructor here unless you're using it for something

Reply:
Thanks for the tip! I thought it was always required.